test/unit/auth/auth.spec.ts

```diff
@@ -1130,6 +1131,84 @@ AUTH_CONFIGS.forEach((testConfig) => {
       });
     });
 
+    describe('getUsers()', () => {
+      let stubs: sinon.SinonStub[] = [];
+
+      afterEach(() => {
+        stubs.forEach((stub) => stub.restore());
+        stubs = [];
+      });
+
+      it('should be rejected when given more than 100 identifiers', () => {
+        const identifiers: UserIdentifier[] = [];
+        for (let i = 0; i < 101; i++) {
+          identifiers.push({uid: 'id' + i});
+        }
+
+        expect(() => auth.getUsers(identifiers))
+          .to.throw(FirebaseAuthError)
+          .with.property('code', 'auth/maximum-user-count-exceeded');
+      });
+
+      it('should return no results when given no identifiers', () => {
+        return auth.getUsers([])
+          .then((getUsersResult) => {
+            expect(getUsersResult.users).to.deep.equal([]);
+            expect(getUsersResult.notFound).to.deep.equal([]);
+          });
+      });
+
+      it('should return no users when given identifiers that do not exist', () => {
+        const stub = sinon.stub(testConfig.RequestHandler.prototype, 'getAccountInfoByIdentifiers')
+          .resolves({});
+        stubs.push(stub);
+        const notFoundIds = [{uid: 'id that doesnt exist'}];
+        return auth.getUsers(notFoundIds)
+          .then((getUsersResult) => {
+            expect(getUsersResult.users).to.deep.equal([]);
+            expect(getUsersResult.notFound).to.deep.equal(notFoundIds);
+          });
+      });
+
+      it('should be rejected when given an invalid uid', () => {
```

Comment:
I think we ought to clearly document this behavior. If any of the identifiers are invalid/malformed, it results in a total failure, as opposed to the partial failure that occurs when some identifiers are not available.